anyway/flask_app.py

```diff
@@ -220,6 +220,16 @@ def markers():
         return generate_json(accident_markers, rsa_markers, discussions, is_thin, total_records=result.total_records)
 
 
+@app.route("/markers_by_yishuv_symbol", methods=["GET"])
+@user_optional
+def markers_by_yishuv_symbol():
+    logging.debug('getting markers by yishuv symbol')
+    yishuv_symbol = request.values.get('yishuv_symbol')
+    markers = db.session.query(AccidentMarker).filter(AccidentMarker.yishuv_symbol == yishuv_symbol).all()
```

Comment:
@advak looks very good. Perhaps it's better to use the AccidentMarkerView (which contains  also fields in Hebrew)?